src/Promitor.Tests.Unit/Agents/ResourceDiscovery/AzureResourceRepositoryTests.cs

```diff
@@ -0,0 +1,256 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json.Linq;
+using Promitor.Agents.ResourceDiscovery.Configuration;
+using Promitor.Agents.ResourceDiscovery.Graph.Interfaces;
+using Promitor.Agents.ResourceDiscovery.Graph.Model;
+using Promitor.Agents.ResourceDiscovery.Graph.Repositories;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Promitor.Tests.Unit.Agents.ResourceDiscovery
+{
+    public class AzureResourceRepositoryTests
```

Comment:
Nit - Can you add the typicaly `Arrange`, `Act`, `Assert` comments like other tests please?

Reply:
Sure thing! Will do.